user_guide/src/introduction.md

```diff
@@ -5,35 +5,34 @@
 This book is an introduction to the
 [`Polars` DataFrame library](https://github.com/pola-rs/polars). Its goal is to
 introduce you to `Polars` by going through examples and comparing it to other
-solutions. Some design choices are introduced here, and the optimal use of `Polars`
-described.
+solutions. Some design choices are introduced here. The guide will also introduce you to
+optimal usage of `Polars`.
 
 Even though `Polars` is completely written in [`Rust`](https://www.rust-lang.org/) (no
-runtime overhead!) and uses [`Arrow`](https://arrow.apache.org/) -the
-[native arrow2 `Rust` implementation](https://github.com/jorgecarleitao/arrow2)- as its foundation, the
+runtime overhead!) and uses [`Arrow`](https://arrow.apache.org/) -- the
+[native arrow2 `Rust` implementation](https://github.com/jorgecarleitao/arrow2) -- as its foundation, the
 examples presented in this guide will be mostly using its higher-level language
-bindings. The latter are merely a thin wrapper that will not offer more
-functionalities than the core library does.
+bindings. Higher-level bindings only serve as a thin wrapper for functionality implemented in the core library.
 
-For people used to [`Pandas`](https://pandas.pydata.org/), the
-[`Python`](https://www.python.org/) bindings are the easiest to get started with
-`Polars`, allowing easier experimentation.
+For [`Pandas`](https://pandas.pydata.org/) users, our
+[Python package](https://pypi.org/project/polars/) will offer the easiest way to get started with
+`Polars`.
 
 ## Goals and non-goals
 
-The goal of `Polars` is being a lightning fast DataFrame library that utilizes all
-available cores on your machine. Unlike tools like dask that tries to parallelize existing single-threaded libraries
-like `numpy` and `pandas`, `polars` is written from the ground up with parallelization of DataFrame queries in mind.
-It goes through great efforts to reduce redundant copies, traverse memory cache efficiently have minimal contention in
+The goal of `Polars` is to provide a lightning fast `DataFrame` library that utilizes all
+available cores on your machine. Unlike tools such as dask -- which tries to parallelize existing single-threaded libraries
+like `NumPy` and `Pandas` -- `Polars` is written from the ground up, designed for parallelization of queries on `DataFrame`s.
+The library emphaizes the objective of reducing redundant copies, thus traversing memory cache efficiently has minimal contention in
```

Comment:
Reducing redundant copies and traversing memory cache efficient are two separate goals. I don't think the sentence should be connected with "thus".

Maybe we should use bullet points as all three objectives are orthogonal.

I also think the "The library emphaizes the objective of reducing" is quite a formal sentence. I like something more informal like "Polars objectives are" or "Polars goes to great efforts to" which is what we do. ;)

Reply:
Hmm I'm a little stuck here. Maybe your bullet list idea would work better, but here's what I put together for that sentence. Lmk what you think.

> `Polars` goes to great lengths to reduce redundant copies while traversing memory cache efficiently, which creates minimal contention in parallelism.

Reply:
No, cache efficiency does not reduce contention. It are separate goals that help performance>

I think it should be:

> Polars goes to great lengths to: reduce redundant copies, traverse memory cache efficiently and minimizing contention in parallelism.

Its a summation of things we focus on. But they are not per se connected.

Reply:
Ahhh. Got it!

Reply:
> minimizing

to

> minimize

cool with you?

Otherwise to make "minimizing" fit we can just "-ing" to the other words.

Reply:
I feel like bullets could work nicely for this too.

> `Polars` goes to great lengths to: 
>  - Reduce redundant copies
>  - Traverse memory cache efficiently 
>  - Minimize contention in parallelism

Now that I understand this better, this sounds like the "trifecta" of `Polars`. So making this super apparent would be cool.

Reply:
Yeap, cool with me. :)